recipes/helm-org

```diff
@@ -0,0 +1,3 @@
+(helm-org :fetcher github
+          :repo "emacs-helm/helm-org"
+          :files ("helm-org.el"))
```

Comment:
This looks like you could use the [default recipe](https://github.com/melpa/melpa#recipe-format) instead of specifying a particular file.

Reply:
We use this at Emacs-helm this for all our Helm external packages, don't know which code is running on Melpa, but I guess specifying a file (for simple package like this) is much faster that listing a directory and extracting the "*.el" files.
If you really want we change this, please let me know.
Thanks.